packages/components/src/components/date-picker/DatePicker.tsx

```diff
@@ -49,13 +49,18 @@ type DatePickerComponentProps = {
   name?: string;
   /** Handle focus event */
   onFocus?: (event: React.FocusEvent<HTMLElement>) => void;
+  /** Handle calendar open event */
+  onCalendarOpen?: () => void;
+  /** Handle calendar close event */
+  onCalendarClose?: () => void;
   placeholder?: string;
   locale?: string;
   placement?: 'top' | 'bottom' | 'right' | 'left';
   todayButton?: string;
   /* The picker is open on render (not supported with menuPortalTarget) */
   showOverlay?: boolean;
   showRequired?: boolean;
+  isResetToValidValue?: boolean;
```

Comment:
I would add a comment about this new prop. Might be confusing to understand what it does 🤔 
The prop name is not clear to me.

Reply:
Hi @antoinerollindev, Could you please have another look at the variable name now that I have updated it?